### DISCUSSION TITLE
Sunshine Windows Service immediately wakes up PC after Sleep post update to v0.18.2

### DISCUSSION THREAD

**Discussion by jhemmond** [LizardByte/community#173] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

After updating to  v0.18.2 yesterday, the Sunshine service is waking up my PC once it goes to Sleep. The same happens when I set it to Hibernate. To verify, I disabled the service, put the PC into Sleep mode, and issue was no longer reproducible.

### Expected Behavior

Windows 11 PC running Sunshine should remain in Sleep/Hibernate mode, and Sunshine service should not wake the PC up. This issues started with  v0.18.2.

### Additional Context

Updating to  v0.18.3 did not fix the issue.

### Host Operating System

Windows

### Operating System Version

Windows 11 10.0.22621 Build 22621

### Architecture

64 bit

### Sunshine commit or version

v0.18.2 and v0.18.3

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

RTX 2080 Super

### GPU Driver/Mesa Version

528.49

**Comment by buthig666** [comment]
Try turning off Hybernate in the advanced power settings. This worked for me.

**Comment by jhemmond** [comment]
I know that will work, but it's not an option for me unfortunately as I need hibernation enabled for my box for other reasons.

My current workaround is to disable timers that wake the PC up from power plan. That put a stop to this problem, and it's a reasonable one. Just wondering why is Sunshine wanting to constantly check for something from Windows, it's just a streaming service.

**Comment by buthig666** [comment]
Oh thank you for replying back.  I too would prefer to use hibernate as well.  I'll give turning off wakeup timer a try.  As long as that doesn't interfere with WOL. 

edit:  I have come to realize that turning off Hibernate isn't a solution.  I have tried every combination of allow hibernate off/on with allow wakeup timers on/off/only important.  Anything work for you w/ permanence?   I just can't seem to figure out how to keep my Pc to sleep when unused.

**Comment by buthig666** [comment, marked as answer]
Ok,  I feel like a nerp.  I currently have allow hibernation and wakeup timers on and have resolved the problem.  In win10 I had to turn off only allow magic packets to wake the computer.  It caused problems for me.  On win11 22h2 now so I turned on that option and everything seems to behave as expected.  The issue has nothing to do with Sunshine.  But setting up Sunshine brought it to my attention.
